### DISCUSSION TITLE
.NET 7 Compatible Release?

### DISCUSSION THREAD

**Discussion by John-MartinMalone** [snoopwpf/snoopwpf#386] [Q&A] [answered]
When will there be a release that works with .NET 7?

**Comment by batzen** [comment, marked as answer]
I just published a preview to chocolatey yesterday which you can install via `choco install snoop --pre` or `choco upgrade snoop --pre`.
Aside from that release you can always grab the latest preview version from https://ci.appveyor.com/project/batzen/snoopwpf/branch/develop/artifacts.

**Reply by John-MartinMalone** [reply]
Thank you 😀
